src/openrct2/world/TileInspector.cpp

```diff
@@ -157,6 +203,22 @@ GameActionResult::Ptr tile_inspector_remove_element_at(const CoordsXY& loc, int1
         {
             return std::make_unique<GameActionResult>(GA_ERROR::UNKNOWN, STR_NONE);
         }
+
+        auto largeScenery = tileElement->AsLargeScenery();
+        if (largeScenery)
+        {
+            // Only delete the banner entry if there are no other parts of the large scenery to delete
+            if (numLargeScenerySequences(loc, largeScenery) == 1)
```

Comment:
Can't test right now, but wouldn't this return 1 and thus remove the banner entry for large banner elements that have been rotated using the tile inspector?

Reply:
Hmm never thought someone would do that.

Reply:
What do you propose as a solution. Should I check all 4 different rotations for the scenery?

Reply:
Ah well thats easy large scenery can't rotate!

Reply:
> What do you propose as a solution. Should I check all 4 different rotations for the scenery?

Yeah, that'd be enough for 2x1 banners. If we later add new banner elements, that'd require another solution. If iterating over all elements on the map is not too expensive, that might be the best thing to do here.

> Ah well thats easy large scenery can't rotate!

Ah, that does sound like a missing feature. For the Goliath letters that would be useful, for instance.